src/victory-container/victory-container.js

```diff
@@ -78,27 +79,40 @@ export default class VictoryContainer extends React.Component {
     return this.timer;
   }
 
+  // overridden in custom containers
+  getChildren(props) {
+    return props.children;
+  }
+
   // Overridden in victory-core-native
   renderContainer(props, svgProps, style) {
-    const { title, desc, children, portalComponent, className } = props;
-    return (
-      <svg {...svgProps} style={style} className={className}>
-        <title id="title">{title}</title>
-        <desc id="desc">{desc}</desc>
-        {children}
-        {React.cloneElement(portalComponent, {ref: this.savePortalRef})}
-      </svg>
-    );
+    const { title, desc, portalComponent, className, standalone } = props;
+    return standalone ?
+      (
+        <svg {...svgProps} style={style} className={className}>
```

Comment:
https://facebook.github.io/react/docs/react-api.html#createelement

Since only the type is different here, this could be refactored by using `React.createElement` directly instead of JSX and then just changing the `type` argument based on the value of `standalone`.